server/welcomebot.go

```diff
@@ -45,6 +45,29 @@ func (p *Plugin) constructMessageTemplate(userID, teamID string) *MessageTemplat
 	return data
 }
 
+func (p *Plugin) constructGlobalMessageTemplate(userID string) *GloablMessageTemplate {
```

Comment:
```suggestion
func (p *Plugin) getGlobalMessageTemplateData(userID string) *GloablMessageTemplate {
```
I think we should rename this